src/main.rs

```diff
@@ -569,6 +569,11 @@ impl Application for Halloy {
 
                     Command::batch(commands)
                 }
+                stream::Update::QuitServer(server, reason) => {
+                    let _ = self.quit_server(server, reason);
+
+                    Command::none()
+                }
```

Comment:
I don't love that we have changed behavior here because we _don't_ return a command.

Reply:
Good point, I think we can just return `self.quit_server(server, reason)` here, which would make more sense.  Works in testing so I'll push that change.  Apologies if I'm misunderstanding the issue.